tests/Oryx.Integration.Tests/Python/PythonSqlServerIntegrationTests.cs

```diff
@@ -27,9 +27,9 @@ public PythonSqlServerIntegrationTests(ITestOutputHelper output) : base(output,
         }
 
         [Theory(Skip = "Bug #1274414")]
-        [InlineData("github-actions")]
-        [InlineData("github-actions-buster")]
-        [InlineData("latest")]
+        [InlineData(ImageTestHelperConstants.GitHubActionsStretch)]
+        [InlineData(ImageTestHelperConstants.GitHubActionsBuster)]
+        [InlineData(ImageTestHelperConstants.LatestTag)]
```

Comment:
nit: looks that we tagged `latest `image to `debian-stretch`, we can change `ImageTestHelperConstants.LatestTag` to `ImageTestHelperConstants.DebianStretch` or `ImageTestHelperConstants.LatestDebianStretch`?